turk-one/.idea/jsLibraryMappings.xml

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
   <component name="JavaScriptLibraryMappings">
+    <file url="PROJECT" libraries="{meteor-packages-auto-import-browser}" />
```

Comment:
@egster I already backed up T2 once, backed it up once more just now. Files are in the shared dropbox folder, and you can use the "mongoimport" command to import the files into your local mongoDB instance. Guide for that is [here](https://docs.mongodb.org/v3.0/reference/program/mongoimport/).